cli/cmd/deploy/deploy.go

```diff
@@ -136,6 +137,18 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 				prodBranch = ghRes.DefaultBranch
 			}
 
+			// check there are no local changes
+			err = isLocalInSyncWithRemote(projectPath, prodBranch)
+			var gitSync *gitSyncError
+			if err != nil && errors.As(err, &gitSync) {
+				msg := gitSync.Error() + ". These changes will not be present in deployed project. Confirm deploy."
```

Comment:
Better to do a `switch` and construct the full output here – makes it easier to understand what the user will see and edit language